src/main/java/software/amazon/documentdb/jdbc/DocumentDbDatabaseMetaData.java

```diff
@@ -308,7 +308,7 @@ public ResultSet getColumns(final String catalog, final String schemaPattern,
             final String tableNamePattern, final String columnNamePattern) throws SQLException {
         final List<List<Object>> metaData = new ArrayList<>();
         if (isNullOrWhitespace(catalog)) {
-            if (isNullOrWhitespace(schemaPattern)
+            if (schemaPattern == null
```

Comment:
The columnNamePattern in addTablesForSchema also needs this change?

Reply:
I have made the change